realm/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -40,9 +40,9 @@
         MEM_ONLY(1);
         //ASYNC(2); // TODO: re-enable when possible
 
-        final int value;
+        public final int value;
```

Comment:
It should be just as readable with proper constants. But no autocompletion help. In this case however, it's internal and has to be converted to a number anyways before passing it to JNI. For external API's I would tend to agree to keep it for anything that's not performance critical.